packages/components/package.json

```diff
@@ -44,6 +44,7 @@
 		"mousetrap": "^1.6.2",
 		"re-resizable": "^6.0.0",
 		"react-dates": "^17.1.1",
+		"react-resize-aware": "^3.0.0-beta.7",
```

Comment:
Relying on a beta probably isn't ideal, but their method of measuring DOM changes is superior to other packages like [`react-use-dimensions`](https://www.npmjs.com/package/react-use-dimensions) because it will trigger an update when content changes outside of a `render()`. Maybe `react-use-dimensions` is enough for this use case 🤷‍♂

Reply:
Apart from window resizing situations (and we know only devs and designers do that, right? 😂 ) is there any real need to update outside of render? If not, `react-use-dimensions` seems perfect for what we need. I'd be reluctant to go the `component-size` route because that polyfill for resize-observer is massive 🙀

Reply:
> Apart from window resizing situations (and we know only devs and designers do that, right? 😂 ) is there any real need to update outside of render?

The update-on-resize is nice, but definitely something we can live without. Worst case, seems like we could apply a `transition: width 0.2s ease;` and at least have the resizing happen in a way that looks less jarring than an instant snap. 👍 👍

Reply:
Its not just window resizing that might cause the dimensions to change. If the width of another element grows or shrinks it might change dimensions of this element without this component having been re-rendered. The use of the `iframe` is a neat way of detecting these layout changes (roughly, when the `iframe` dimensions change it fires the resize event on the `iframe` window).

Given the multi-column layouts in the above screenshots, I think this case is probably something that we need to account for?

Reply:
> Its not just window resizing that might cause the dimensions to change.

This is a good point I had missed. Here's a use-case:

![responsive](https://user-images.githubusercontent.com/1204802/70787009-2154f980-1d8e-11ea-8989-168b1bff3bca.gif)

Reply:
I'll reach out to the developer and see how we might be able to help.

Reply:
@jasmussen The author has released a new version for us! Try bumping the version to `^3.0.0`.